tactics/tacticals.ml

```diff
@@ -224,7 +224,10 @@ let tclDO n t =
     if k < 0 then user_err
       (str"Wrong argument : Do needs a positive integer.");
     if Int.equal k 0 then tclIDTAC
-    else if Int.equal k 1 then t else (tclTHEN t (dorec (k-1)))
+    else if Int.equal k 1 then t
```

Comment:
Don't fix this one, it's dead code in master. It's scheduled for removal as soon as 8.16 is branched.

Reply:
That's no reason to revert the fix imo

Reply:
The rebase will be non-trivial, but if you insist...